src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js

```diff
@@ -17,10 +17,12 @@ const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
 
 export const buildTimeEntriesFilter = (date, userId) => {
   const startDateFilterCondition = joinAndConditions(
+    "ownerId=='" + userId + "'",
     ...filterTimeEntriesOnDate('actualStartTime', date)
```

Comment:
Not sure if the line breaks will go weird but I’m a fan of template strings over concatenation. Would this work?
```suggestion
    `ownerId=='${userId}'${...filterTimeEntriesOnDate('actualStartTime', date)}`
```

Reply:
Not sure the above would work. We need to add the && in-between filter arguments. I have pushed a changed that stores "ownerId=='" + userId + "'" as a const. As this is used multiple times.